### PR TITLE
Pin click to run black

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

**Comment by bmbouter** [comment]
I think this is right. Do we have an issue open with the click community tracking their resolution of it?

**Comment by fao89** [comment]
> I think this is right. Do we have an issue open with the click community tracking their resolution of it?

we have it on black:
https://github.com/psf/black/issues/2964

File: .github/workflows/bootstrap.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Lint plugin_template
         working-directory: plugin_template
         run: |
-          pip3 install black flake8
+          pip3 install black flake8 "click<8.1"
```
Comment (bmbouter) [inline .github/workflows/bootstrap.yaml:30]: Wonderful. Can you put that link in the comment of this line and also the line below it. Something like `Unpin click when <link here> is resolved.

**Review by bmbouter — APPROVED** [review]
Thank you!

**Pull request merged**